openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -79,9 +79,16 @@ def create(self, subset_name, instance_data, pre_create_data):
                 "review_width": preset["Resolution"]["width"],
                 "review_height": preset["Resolution"]["height"],
                 "isolate": preset["Generic"]["isolate_view"],
-                "imagePlane": preset["Viewport Options"]["imagePlane"],
                 "panZoom": preset["Generic"]["pan_zoom"]
             }
+
+            if "Viewport Options" in preset:
+                mapping.update({
+                    "imagePlane": preset["Viewport Options"]["imagePlane"]})
+            elif "ViewportOptions" in preset:
+                mapping.update({
+                    "imagePlane": preset["ViewportOptions"]["imagePlane"]})
```

Comment:
Can we add a comment to state which is the deprecated one? Having a `# TODO: Deprecated after removal of X` type of comment can also help finding these spots that need fixing later.

Reply:
added

Reply:
I don't see it in the changes? :) Did you push it?

Reply:
It's my bad and we find out that it is not really related to creator settings but the conversion of the code to AYON settings.

Reply:
Great - so we can remove this logic / close the PR and fix it differently?

Reply:
> Great - so we can remove this logic / close the PR and fix it differently?

yes i will close it first and then make another PR for fix.(not a good branch name)